contracts/src/ethereum/TokenBridge.sol

```diff
@@ -133,16 +137,17 @@ contract TokenBridge is
 
     function sendMessageStaticAToken(address l1Token, uint256 accRewards) 
```

Comment:
Who is allowed to call this function?

Reply:
Yeah or use `msg.sender` instead allowing an explicit address to be passed in